### DISCUSSION TITLE
Manage BackGroudServer when we scale Horizontally

### DISCUSSION THREAD

**Discussion by dilip-dhankecha** [jobrunr/jobrunr#1001] [Q&A]
In backGround server, we have the worker count configurations which we can configure. So let's assume we have total 3 Jobs in our spring boot application which is really a long running process and the worker count is set to 2.
Now when we increase the new instance then we have 2 background server which will handle the distributing of the jobs. How it will manage the distribution of the job?
How jobrunr identify that how many application is running when we scale horizontally? Here each instance up the new Background job server that is fine?
How many worker node we have to use when we have 3 job to process?

**Comment by rdehuyss** [comment]
@dilipdhankecha2530 this is all documented or available on youtube.
